bin/sirius-script-rabpm-mngr.py

```diff
@@ -95,6 +108,8 @@ def main():
         run_ioc_restart(args.rack, args.ioc, args.slot)
     elif args.command == "rffe-reset":
         run_rffe_reset(args.rack, args.vslot)
+    elif args.command == "boot-from-flash":
```

Comment:
Boot from flash seems like a very technical term. I think we can use something else with the same meaning. `afc-reset`?